examples/hotrod/pkg/tracing/init.go

```diff
@@ -43,9 +44,28 @@ import (
 
 var once sync.Once
 
-// Init initializes OpenTelemetry SDK and uses OTel-OpenTracing Bridge
+// InitOTEL initializes OpenTelemetry SDK
+// to return an Otel tracer.
+func InitOTEL(serviceName string, exporterType string, metricsFactory metrics.Factory, logger log.Factory) trace.Tracer {
+	_, oteltp := initBOTH(serviceName, exporterType, metricsFactory, logger)
+	otel.SetTracerProvider(oteltp)
```

Comment:
If we can get away with it, I would prefer not to set this global. Right now you certainly don't need it, so let's see how other OTEL instrumentation is done in the later PRs.

Reply:
Thanks for that.